src/repositories/sql/users/update_avatar.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE users
+SET
+avatar_image=$1,
+updated_at= (SELECT NOW())
+WHERE user_id=$2
+RETURNING user_id;
```

Comment:
- Add indentations for better readability
```
UPDATE users
  SET
    x=y
  WHERE ...
```
- You can just write `updated_at = NOW()`
- Make spacing consistent, pick either `x = y` or `x=y`, but not both